src/AdminSettings.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Drupal\media_webdam;
+
+use cweagans\webdam\Exception\InvalidCredentialsException;
+
+/**
+ * Class AdminSettings.
+ *
+ * Gets Webdam folders into config form as options.
+ *
+ * @package Drupal\media_webdam
+ */
+class AdminSettings {
```

Comment:
Please remove this class and the associated services.yml entry. Just inject the Webdam service directly - no need to wrap the call in this class.